discord.go

```diff
@@ -74,6 +74,7 @@ func New(args ...interface{}) (s *Session, err error) {
 	s.Identify.GuildSubscriptions = true
 	s.Identify.Properties.OS = runtime.GOOS
 	s.Identify.Properties.Browser = "DiscordGo v" + VERSION
+	s.Identify.Intents = 32767
```

Comment:
This value `32767` seems a bit like a magic number. How was it obtained? Can consts be used to define/build up to it? Do we also want it to be the default value rather than zero?

Reply:
We can't use 0 cause no event will be received from discord - it will basicly 'broke' old implementation. 

32767 means all intents so it's eq to `1 << 0 | 1 << 1 | 1 << 2 | 1 << 3 | 1 << 4 | 1 << 5 | 1 << 6 | 1 << 7 | 1 << 8 | 1 << 9 | 1 << 10 | 1 << 11 | 1 << 12 | 1 << 13 | 1 << 14`

https://discordapp.com/developers/docs/topics/gateway#list-of-intents

I can move it to const as well.